addons/payment_paypal/README.md

```diff
@@ -0,0 +1,17 @@
+# PayPal
+
+## Technical details
+
+API: [PayPal Payments Standard](https://developer.paypal.com/api/nvp-soap/paypal-payments-standard/integration-guide/formbasics/)
+
+This module integrates PayPal using `payment`'s form-based implementation of the payment with
+redirection flow.
+
+## Supported features
+
+- Payment with redirection flow
+- Webhook notifications
+
+## Testing instructions
+
+Payments must be made using a separate [sandbox account](https://www.sandbox.paypal.com/myaccount/).
```

Comment:
maybe we can give a link to instructions https://developer.paypal.com/tools/sandbox/ rather than to the login page?

Reply:
I added both :)